README.md

```diff
@@ -289,7 +317,10 @@
     var superPower = new SuperPower();
     ```
 
-  - Use multiple `var` declarations for multiple variables and declare each variable on a newline. Justification: http://benalman.com/news/2012/05/multiple-var-statements-javascript
+  - Use one `var` declaration per variable.
```

Comment:
Hrm... I like the direction here, but, if quickly scanning someone could easily stop after "declaration" and think you just use _one_ var per function. Maybe "Each variable needs it's own `var`" or "Use a `var` for each variable". Whatever it is, it needs to be a bit more scan-proof.